core/lib/spree/core/engine.rb

```diff
@@ -272,6 +276,10 @@ class Engine < ::Rails::Engine
         Rails.application.config.spree.analytics_event_handlers = []
 
         Rails.application.config.spree.integrations = []
+
+        Rails.application.config.spree.address_validators = [
+          Spree::Addresses::PhoneValidator
+        ]
```

Comment:
@brozek95 let's make this structure as `validators.address` so we can later add them for products, and so on